### DISCUSSION TITLE
Positive grade

### DISCUSSION THREAD

**Discussion by Stieguu** [sebastian-hofstaetter/teaching#64] [General]
Did I understand correctly, that if we did not receive >= 50 % on the first exercise we cannot receive a positive grade this semester?

Thank you and BR,
Alex Stieger

**Comment by sebastian-hofstaetter** [comment]
Hi,
Yes that is correct, although if there is any reason why you couldn't finish and you want to continue this course you can send us an email to the course-address and we can work out an additional task you could do to fill in the gap.
Best,
Sebastian
